printer/printer_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package printer
+
+import (
+	"bufio"
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type mockWriter struct {
+	buffer []byte
+}
```

Comment:
We can directly create a struct with just the `[]byte` type avoiding referring to the `buffer` property:

```suggestion
type mockWriter []byte
```

Reply:
Good point @mgdelacroix! Thanks 👍